lib/spack/spack/cmd/doc.py

```diff
@@ -24,6 +24,8 @@
 ##############################################################################
 
 description = "run pydoc from within spack"
```

Comment:
Ooh, this is very interesting. Yeah, `spack pydoc` sounds better to me. Can you add documentation on both of these commands to the Developer Guide? Just a single paragraph blurb and an example for running `spack pydoc`.